src/worker/lambda/lambdaFunction.go

```diff
@@ -303,9 +303,9 @@ func (f *LambdaFunc) Task() {
 
 		// AUTOSCALING STEP 1: decide how many instances we want
 
-		// let's aim to have 1 sandbox per second of outstanding work
+		// let's aim to have 1 sandbox per 10ms of outstanding work
 		inProgressWorkMs := outstandingReqs * execMs.Avg
-		desiredInstances := inProgressWorkMs / 1000
+		desiredInstances := inProgressWorkMs / 10
```

Comment:
We talked about this before. I think either the policy itself should be configurable, or we should be able to set this parameter instead of hardcoding it.

Reply:
Sounds good!  Should we add a TODO that we should make it configurable at some point?